packages/block-library/src/navigation/edit/deleted-navigation-warning.js

```diff
@@ -2,12 +2,29 @@
  * WordPress dependencies
  */
 import { Warning } from '@wordpress/block-editor';
-import { Button } from '@wordpress/components';
+import { Button, Notice } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { createInterpolateElement } from '@wordpress/element';
 
-function DeletedNavigationWarning( { onCreateNew } ) {
-	return (
+function DeletedNavigationWarning( { onCreateNew, isNotice = false } ) {
+	return isNotice ? (
+		<Notice status="warning" isDismissible={ false }>
+			{ createInterpolateElement(
```

Comment:
To avoid code duplication, the whole `createInterpolateElement()` should be assigned to a `const` that is then used for both the Notice and the Warning.